phpcs.xml

```diff
@@ -61,4 +61,8 @@
 		<exclude-pattern>tests/php</exclude-pattern>
 		<exclude-pattern>tests/Tools/</exclude-pattern>
 	</rule>
+
+	<rule ref="WordPress.NamingConventions.ValidFunctionName.MethodNameInvalid">
```

Comment:
But if we are going PSR4, why not going fully? We are already using Pascal case for class names anyway.

Reply:
There's a point to be made about consistency in code style across the organization, but to be fair, camelCase is used in JavaScript! This is more a decision to be made as a team though.

Reply:
I definitely agree with the sentiment that it helps shift perspective though!

Reply:
As discussed in the team call, we're going to stick to `camel_case` for method names.

Reply:
@Konamiman @ObliviousHarmony if there was a decision to keep using `snake_case` for method names (I'm assuming that is what @Konamiman meant by `camel_case`), shouldn't this rule be removed? It seems it got merged to master: https://github.com/woocommerce/woocommerce/blob/eb65252ba8de26edfa5be84a452d33d1ffd04880/phpcs.xml#L101

Reply:
Yep, that seems to be the case. I'll remove this in the PR to reinstate the dependency injection engine.

Reply:
@Konamiman maybe this can go on a separate PR? I think removing this is pretty straigthforward and the DI engine PR is a big that might take sometime to be merged.